src/cmdstan-guide/installation.Rmd

```diff
@@ -1,6 +1,38 @@
 # CmdStan Installation
 
-To install CmdStan you need:
+## Installation via `conda`
+
+CmdStan can be installed via the package management system [conda](https://docs.conda.io/en/latest/)
+via the [conda-forge channel](https://conda-forge.org/). This
+will install a pre-built version of CmdStan along with the required
+dependencies (i.e. a C++ compiler, a version of Make, and required
+libraries) detailed below under [Source installation].
+
+One can create a new conda environment and install CmdStan with
+the following command.
+
+```
+conda create -n cmdstan -c conda-forge cmdstan
+```
+or install it in an existing environment with
+```
+conda install -c conda-forge cmdstan
+```
+
+With the conda installation, one can use the R or Python
+bindings to CmdStan, or can use the command `cmdstan_model`
+to activate the CmdStan makefile from anywhere. To enable either
+of these, ensure your environment is activated (via
+`conda activate cmdstan`, or whichever name you used above).
```

Comment:
It helps to have a name for the name we're talking about here, like "package identifier" or whatever conda uses.  Then you can replace "name" with "package identifier" and it'll be clearer.  Then you can write `conda activate <package-id>` and say `<package-id>` is `cmdstan` or whatever other identifier you chose.